### ISSUE TITLE
Binary operator '|=' cannot be applied to two 'UIAccessibilityTraits' operands in swift 4.2

### ISSUE THREAD

**Issue by PramodShukla10** [xmartlabs/XLPagerTabStrip#627]
Before submitting issues ...

- Make sure you are using XLPagerTabStrip [latest release](https://github.com/xmartlabs/XLPagerTabStrip/releases) or master branch version.
- Make sure your Xcode version is the latest stable one.
- Check if the issue was [already reported or fixed](https://github.com/xmartlabs/XLPagerTabStrip/issues?utf8=%E2%9C%93&q=is%3Aissue). We add labels to each issue in order to easily find related issues. If you found a match add a brief comment "I have the same problem" or "+1".
- Please do not use the issue tracker for personal support requests. Stack Overflow is a better place for that where a wider community can help you!

When submitting issues, please provide the following information to help maintainers to fix the problem faster:

- Environment: XLPagerTabStrip, Xcode and iOS version you are using.
- In case of reporting errors, provide Xcode console output of stack trace or code compilation error.
- Any other additional detail such as example code that you think it would be useful to understand, reproduce and solve the problem.

**Comment by henryw14** [comment]
I also encountered this error when pulling the 8.0.1 release. Updating my pod to:
`pod 'XLPagerTabStrip', :git => 'https://github.com/xmartlabs/XLPagerTabStrip', :branch => 'master'`

fixed the problem. This should stopgap for anyone else until the next release

**Comment by razelkayamdev** [comment]
@henryw14 Thanks!

**Comment by mats-claassen** [comment]
Fixed in 8.1.0
